controllers/operands/kubevirt.go

```diff
@@ -136,6 +139,7 @@ var (
 		kvHotplugNicsGate,
 		kvVMPersistentState,
 		kvHNetworkBindingPluginsGate,
+		kvAlignCPUsGate,
```

Comment:
A FG that is added here implies that the U/S FG has been graduated or is scheduled to graduation very soon (and there is commitment that it will actually happen).

If that is not the case, then we have a problem of too early adoption.
I would expect HCO to track the graduation & lifecycle of the U/S project, as setting the FG by default is in practice declaring it GA.

If the intention is for the user not to be bothered, then the U/S should graduate the feature first.
If the intention is to release it at tech-preview, then the FG needs to be defaulted to "false" and exposed through the HCO config.

Please avoid hacking the FG system to reduce UX friction on D/S. The problem is on U/S and there the problem should be solved.

Reply:
Thank you for this important notice.
@dankenigsberg - keep me honest her.

The FG we are introducing as default enabled is so that users can run DPDK VMIs. The behavior this FG is setting is of very low risk, so I believe we are OK with setting it ON on D/S. 
In the future we may need to change kubevirt API and eventually remove this FG. IIUC this can done by both removing the FG on Kubevirt and here, thus making the behavior constant for both U/S and D/S.

Reply:
@RamLavi if you have to expose the feature gate here in HCO, that would be sad, but acceptable. Make sure to document it very clearly.

Reply:
@dankenigsberg I'm afraid I don't follow you. Do you mean we should add a "HCO FG/enabler" that would set the annotation+FG on kubevirt CR (like we originally did)?

Reply:
I don't know what you did originally (I saw this PR only today). But yes, if you think this feature is a risky and think that it may still be reverted, you can add a "HCO FG/enabler" that would set the annotation+FG on kubevirt CR.

Reply:
As we discussed offline yesterday - there is no real need for such an "enabler" from HCO: most users won't notice this change, and those that do and really oppose can choose the jsonPatch solution in order to remove the annotation.

Reply:
I am happy about not tiring HCO users about this kubevirt detail.

Reply:
Are json patches generally viable? I think their usage triggers some kind of alert.

Reply:
they are not. I was only mentioning that if all goes wrong - we should make sure that they are able to save us (currently they are agnostic to kubevirt CR annotations for obvious reasons)